doc/development/plugins.rst

```diff
@@ -371,8 +373,8 @@ where
   which is the simple expectation value of the operator :math:`\hat{B}` evolved via the gate
   :math:`O(\phi_k)`.
 
-Note that if ``grad_recipe = None``, the default gradient recipe is
-:math:`(c_k, s_k)=(1/2, \pi/2)` for every parameter.
+  Note that if ``grad_recipe = None``, the default gradient recipe is
+  :math:`(c_k, s_k)=(1/2, \pi/2)` for every parameter.
 
 The user can then import this operation directly from your plugin, and use it when defining a QNode:
```

Comment:
```suggestion
The user can then import this operation directly from your plugin, and use it when defining a :meth:`~.QNode`:
```

Reply:
Same comment here, there isn't a good candidate for this reference. In a lot of the documentation, we tend to use 'QNode' as a noun without referencing